libs/langchain/langchain/chat_models/vertexai.py

```diff
@@ -91,10 +100,23 @@ def _parse_examples(examples: List[BaseMessage]) -> List["InputOutputTextPair"]:
     return example_pairs
 
 
+def _get_question(messages: List[BaseMessage]) -> HumanMessage:
+    """Get the human message at the end of a list of input messages to a chat model."""
+    if not messages:
+        raise ValueError("You should provide at least one message to start the chat!")
+    question = messages[-1]
+    if not isinstance(question, HumanMessage):
```

Comment:
Are there no situations when we want to continue generation given the last message was actually AI? Or maybe it was just the system message?

Reply:
cc @lkuligin

Reply:
I doubt it's the case for a chat model. And Vertex chat model exposes a `send_message` method, so you can't actually send an AI message there.
https://cloud.google.com/python/docs/reference/aiplatform/latest/vertexai.language_models.ChatSession#vertexai_language_models_ChatSession_send_message